test/sim/moves/futuresight.js

```diff
@@ -85,11 +85,12 @@ describe('Future Sight', function () {
 		battle.makeChoices();
 		battle.makeChoices();
 		battle.makeChoices();
+		battle.makeChoices();
```

Comment:
I thought Future Sight hit at the end of the third turn, so you shouldn't need a fourth turn here. Are you sure you're really fixing the bug, and not just giving Wynaut a second chance to set Future Sight?

Reply:
This is 3 turns. The third choice seems to be switching to Roggenrola after Shedinja faints.

Reply:
Ah, in that case I think a comment on the switch choice would make this clearer.

Reply:
resolved

Reply:
Thanks!